### DISCUSSION TITLE
Model .select method seems to be escaping characters

### DISCUSSION THREAD

**Discussion by MateoSRQ** [adonisjs/core#4489] [Help]
In order to make dynamic column based on JSONB, SQL like this is needed

`SELECT *, properties->>'$.ImageSource' as  ImageSource FROM images;`

However, produced query seems like

``properties->>'$`.`fill'``

Which fails, because is enclosed in ticks and also the point is enclosed.

It is any way to work around this?

Thanks

**Comment by holoyan** [comment]
maybe ` db.raw` ?

```typescript

db
  .from('users')
  .select(
    db.raw(`
      (select ip_address from user_logins where users.id = user_logins.user_id limit 1) as last_login_ip
    `)
  )


```


[docs](https://lucid.adonisjs.com/docs/select-query-builder#select)
